README-ja.md

```diff
@@ -0,0 +1,75 @@
+# The Kubernetes documentation
```

Comment:
nit: This title is in English.

Reply:
I did it on purpose, should I translate it as well?

Reply:
I saw that other localized READMEs have the title translated, so it can be about following a majority pattern.

Reply:
@sftim Aight fixed.